nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/nf/canvas/nf-actions.js

```diff
@@ -845,6 +845,116 @@
             }
         },
 
+        /**
+         * Enable all controller services in the specified ProcessGroup.
+         *
+         * @argument {selection} selection      The selection
+         * @argument {cb} callback              The function to call when request is processed
+         */
+         enableAllControllerServices: function (selection,cb) {
```

Comment:
What is `cb` used for in this case? Other actions (like `stop` and `terminate`) had a callback introduced when we introduced the ability to stop a running Processor from the details dialog and subsequently open the configuration dialog (NIFI-5986). I don't see a similar capability here with enabling a Controller Service.

Reply:
Callback function removed.